### ISSUE TITLE
Login final req's for gui

### ISSUE THREAD

**Issue by sellnat77** [CECS343Project/Farm#33]
Put "username" and "Password" as the default text in the textboxes for the login should be 

``` java
TextBox tbUserName = new TextBox();
TextBox tbPassword = new TextBox();

tbUserName.setText("UserName");
tbPassword.setText("Password");
```

For Sign up, change the date of birth selection to datePicker widget:

``` java
 DatePicker datePicker = new DatePicker();
```

Change suffix to a list box

**Comment by sellnat77** [comment]
``` java
TextBox box = new TextBox();
box.setText("Default Text");
box.addKeyboardListener(this);
defaultValue = true; // this is a global boolean value
```

Then have your class implement KeyboardListener leaving them all blank except:

``` java
public void onKeyPress(Widget arg0, char arg1, int arg2) 
{
    if(defaultValue)
    {
        box.setText = "";
        defaultValue = false;
    }
}
```

**Comment by sellnat77** [comment]
c0afd02c2c87335dff7c5ffd8d1c0c9865500b65 fixes #33
